lib/utils/isKeyframeSelector.mjs

```diff
@@ -1,4 +1,9 @@
-import { keyframeSelectorKeywords } from '../reference/keywords.mjs';
+import { keyframeSelectorKeywords, namedTimelineRangeKeywords } from '../reference/keywords.mjs';
+
+const HAS_TIMELINE_RANGE = new RegExp(
+	`^(?:${[...namedTimelineRangeKeywords.values()].join('|')})\\s+(?:\\d+|\\d*\\.\\d+)%$`,
```

Comment:
note

This requires the selector to be trimmed before being passed.

Reply:
Correct, like the other checks in this function